easybuild/easyconfigs/g/GDAL/GDAL-2.2.0-intel-2017a-Python-2.7.13-HDF5-1.10.1-HDF-4.2.13.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '2.2.0'
+hdf5_ver = '1.10.1'
+hdf_ver = '4.2.13'
+versionsuffix = '-Python-%%(pyver)s-HDF5-%s-HDF-%s' % (hdf5_ver, hdf_ver)
+
+homepage = 'http://www.gdal.org/'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://download.osgeo.org/gdal/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['0d4c326862e0f118e17418c042c2bcd037b25abd3fb198e1fc5d40b11a9fc8ea']
+patches = ['GDAL-%(version)s_fix-uchar-definition.patch']
```

Comment:
Move up `patches` above `checksums`, also include a checksum for the patch?

You can let `eb --inject-checksums` do that for you, see http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#adding-or-replacing-checksums-using-inject-checksums.

Reply:
Did a manual job to update the checksums